stylesheets/partials/page-specific/pets-list.scss

```diff
@@ -0,0 +1,52 @@
+// TODO - MISSING COLOR in Variables
+$gray: #808080;
```

Comment:
To see this in action, I am just replacing the existing file that is in Site Executive, with the dist version generated in my bracnh and swapping it out in the chrome console.  

See [Adoptable Pets](http://www.baltimorecountymd.gov/Agencies/animalservices/adoption.html) for usage.